src/azure-cli-core/azure/cli/core/_help.py

```diff
@@ -44,8 +44,13 @@
 
 Welcome to the cool new Azure CLI!
 
-Use `az --version` to display the current version.
-Here are the base commands:
+Documentation: https://docs.microsoft.com/en-us/cli/azure
+GitHub repo: https://github.com/Azure/azure-cli
+
+Use `az --version` to display the current version in human-readable format, `az version` to display the current version in JSON format.
```

Comment:
is it necessary to put this detail in the main message?

Reply:
I am not sure. Maybe we can totally remove this line.